src/main/java/org/traccar/protocol/LaipacProtocolDecoder.java

```diff
@@ -230,12 +253,17 @@ protected Object decodeAvrmc(
 
         String event = parser.next();
         position.set(Position.KEY_ALARM, decodeAlarm(event));
-        position.set(Position.KEY_EVENT, decodeEvent(event, position));
+        position.set(Position.KEY_EVENT, decodeEvent(event, position, deviceModel));
         position.set(Position.KEY_BATTERY, Double.parseDouble(parser.next().replaceAll("\\.", "")) * 0.001);
         position.set(Position.KEY_ODOMETER, parser.nextDouble() * 1000);
         position.set(Position.KEY_GPS, parser.nextInt());
         position.set(Position.PREFIX_ADC + 1, parser.nextDouble() * 0.001);
-        position.set(Position.PREFIX_ADC + 2, parser.nextDouble() * 0.001);
+
+        if ("AVL110".equals(deviceModel) || "AVL120".equals(deviceModel)) {
+            position.set(Position.PREFIX_ADC + 2, parser.nextDouble() * 0.001);
+        } else {
+            parser.next();
+        }
```

Comment:
Why are we changing this?

Reply:
> Why are we changing this?

There are models not having analog input 2 (SF-Lite for example). I don't know about other models (beside AVL110/120) that has it. Right now I don't remember if I wrote that part myself in the past, but that time, I didn't see the need to use device model to conditionally set analog input 2.

Reply:
What do they report? Empty or actual value? I see some empty fields in the documentation.

Reply:
SF-Lite, always reports a zero for analog input 2, but physically, it lacks such input. Uploaded documentation is really old (I don't have recent one) but firmware is more recent. We can actually condition setting analog input 2 according to the field being empty or not, but, as I said, and as currently, we will see/store info about it even if the device lacks it. You tell me...